Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -259,13 +272,13 @@ extension Module  {
                 // example would be `@executable_path/../lib` but there are
                 // other problems to solve first, e.g. how to deal with the
                 // Swift standard library paths).
-                buildSettings["LD_RUNPATH_SEARCH_PATHS"] = buildSettings["LD_RUNPATH_SEARCH_PATHS"]! + " @executable_path"
+                buildSettings["LD_RUNPATH_SEARCH_PATHS"] = [toolchainPath, "@executable_path"]
```

Comment:
It's actually better to keep this as appending to the value of the `LD_RUNPATH_SEARCH_PATHS` setting, since that is the semantic action here.  It happens that the toolchain path is the thing that's assigned above, so right now it works out the same in practice, but if anyone adds code in between to add something else to `LD_RUNPATH_SEARCH_PATHS`, that would end up getting stomped by this line of code.